src/helpers/state.js

```diff
@@ -1,20 +1,20 @@
-'use strict';
+'use strict'
 
-let _ = require('lodash');
+let _ = require('lodash')
```

Comment:
noticed a few `let`s in this source code that should be `const` but not introduced by you